docs/_includes/layouts/pages/basic.njk

```diff
@@ -5,6 +5,7 @@ importElements:
   - rh-footer
   - rh-alert
   - rh-subnav
+  - rh-table
```

Comment:
Instead of adding `rh-table` to the page level layout, please add It to the individual pages in the front matter.

`foundations/typography/index.md` & `foundations/typography/index.md`
```
importElements:
  - rh-table
```

Otherwise, we are polluting table JS across all pages that might not use it.  It is also true that `rh-alert` should be evaluated in this very same way here (which was not added by you).  basic should be kept to the lowest common denominator used in all pages.